popSimManu.tex

```diff
@@ -145,15 +145,24 @@ \section*{Introduction}
 benchmark the quality of inferences gleaned from computational population genetics.
 As large databases of population genetic variation begin to be used to inform public health procedures,
 the accuracy and quality of these inferences is becoming ever more important.
+%ALG: cite large databases of population genetic variate to inform public health procedures? UK biobank, 23andMe???
```

Comment:
I think this is well-known enough so that we don't need to clutter the text with a list of citations. But if you know a good review of the topic (prospectus on future PRS score use?) then that'd be a useful thing to cite here?

Reply:
that's fine. Not off the top of my head!